runtime/proto/corfu_options.proto

```diff
@@ -19,6 +19,16 @@ message SchemaOptions {
     repeated string stream_tag = 5;
     // Nested Secondary Key (repeated field)
     repeated SecondaryIndex secondary_key = 6;
+    // The combination of client and logical_group name to indicate how data is to be replicated.
+    optional ReplicationLogicalGroup replication_group = 7;
+}
+
+message ReplicationLogicalGroup {
+    // Logical group name that helps club a number of tables into one group for Log Replication.
+    required string logical_group = 1;
```

Comment:
is 'required' just informational? or we should see an error if replication_group is defined without one of these fields? if the last is the case, could we have a test or would it fail on compile time?

Reply:
We cannot add a test case since the proto will not compile without both fields specified
we will see errors like these `sample_schema.proto:81:67: Error while parsing option value for "replication_group": Message missing required fields: client_name`